index.html

```diff
@@ -0,0 +1,105 @@
+      <div class="top__logo">
+        <img src="" alt="Logo" class="top__logo__img" />
+      </div>
+      <nav>
+        <ul class="top__nav">
+          <li><a href="#about">About Me </a></li>
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#stack">Tech Stack</a></li>
+          <li><a href="#cv">Links of Interest</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section id="about">
+        <div class="about__hero">
+          <img class="about__hero__img" src="" alt="" />
+        </div>
+        <div class="about__content">
+          <h1>Moin Moin und Hallo</h1>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Laboriosam,
+            doloremque! Officia distinctio dolores rerum omnis unde
+            consequuntur. Fugiat tempora, soluta possimus quas eum quia
+            excepturi quo aliquam vero provident iste officia magni dolorem amet
+            veniam magnam. Tempore laudantium aperiam impedit, expedita earum
+            non quos assumenda delectus deleniti laborum nemo, aliquid
+            doloremque quo et pariatur accusamus numquam quibusdam fugiat dolor
+            corrupti dignissimos architecto! Placeat praesentium debitis
+            suscipit maiores voluptatem laborum perferendis, unde accusamus vel
+            ullam ipsam nobis dignissimos tempore voluptatibus quia excepturi
+            tenetur magni illo. Quaerat, voluptate eaque! Ut laboriosam aliquam
+            incidunt voluptas? Eius quam illum, ea beatae in vitae quidem dicta
+            commodi! Et excepturi accusamus dicta assumenda mollitia nostrum
+            tempora ullam, distinctio voluptatum optio maiores modi deserunt.
+            Nulla, numquam quasi!
+          </p>
+        </div>
+      </section>
+
+      <section id="portfolio">
+        <div class="portfolio__card">
+          <img class="portfolio__card__img" src="" alt="Hero" />
+        </div>
+        <div class="portfolio__card__content">
+          <h2>Portfolio Headline</h2>
+          <ol>
+            <li>Projekt 1</li>
+            <li>Projekt 2</li>
+            <li>Projekt 3</li>
+            <li>Projekt 4</li>
+            <li>Projekt 5</li>
+          </ol>
+        </div>
+      </section>
+
+      <section id="stack">
+        <div class="stack__card">
+          <img class="stack__card__img" src="" alt="" />
+        </div>
+        <div class="stack__card__content">
+          <h2>Stack Timeline</h2>
+          <ul>
+            <li>Projekt 1</li>
+            <li>Projekt 2</li>
+            <li>Projekt 3</li>
+            <li>Projekt 4</li>
+            <li>Projekt 5</li>
+          </ul>
+        </div>
+      </section>
+
+      <aside id="cv">
+        <div>
+          <button>Lebenslauf</button>
+        </div>
+
+        <div>
+          <button>Zertifikate</button>
+        </div>
+        <div>
+          <button>Anschreiben</button>
+        </div>
```

Comment:
Do you need the divs?

Reply:
I thought about adding additional icons just before the buttons, 
that's why I wrapped them in divs already. 
I am unsure wether or not to use emojis as p or actual png/svg icons

Reply:
You can use emojis or svg icons. That is up to you. But is a bit like this example here:
https://github.com/me-jterhorst/my-personal-website/pull/1#discussion_r653690645